pdns/gss_context.cc

```diff
@@ -0,0 +1,430 @@
+#ifdef HAVE_CONFIG_H
```

Comment:
I know we haven't been doing this too often in the past, but is it possible to have a small docblock for these functions that describes their purpose?